### PR TITLE
Update container version when testing.

### PR DESCRIPTION
**Pull request by YoEight**
Changed: Update container version when testing.

### CODE REVIEW

File: .github/workflows/rust.yml [hunk 1]
```diff
@@ -33,7 +33,7 @@ jobs:
     needs: build
     strategy:
       matrix:
-        version: [ 20.10.5-focal, 21.10.0-focal, ci ]
+        version: [ previous-lts, lts, latest ]
```
Comment (thefringeninja) [inline .github/workflows/rust.yml:36]: why `latest` and not `ci`? `latest` is going to be the same as `lts` most of the time

Comment (YoEight) [reply .github/workflows/rust.yml:36]: It seems the `ci` tag is no longer updated unlike `latest`

Comment (pvanbuijtene) [reply .github/workflows/rust.yml:36]: CI is only build from master, but something is wrong with the tags...looking into it, so think we should stick to `ci`.

Comment (YoEight) [reply .github/workflows/rust.yml]: Done

**Review by pvanbuijtene — APPROVED** [review]

**Pull request merged**